responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/household-members/household-members.component.ts

```diff
@@ -177,13 +173,20 @@ export class HouseholdMembersComponent implements OnInit, OnDestroy {
   deleteRow(index: number): void {
     this.dialog
       .open(DialogComponent, {
-        data: { component: DeleteHouseholdDialogComponent },
+        data: {
+          component: YesNoDialogComponent,
+          title: 'Remove Household Member',
+          text:
```

Comment:
Create dialog object to set these values. No static value should be in the components. Add all static values to global constants